src/Geometry.cpp

```diff
@@ -643,6 +647,27 @@ Point* Geometry::getFSRPoint(int fsr_id) {
 
 
 /**
+ * @brief Return the centroid for a given FSR ID
+ * @param fsr_id the FSR ID
+ * @return the FSR's centroid
+ */
+Point* Geometry::getFSRCentroid(int fsr_id) {
+
+  Point* point;
+
+  try{
+    point = _FSR_keys_map.at(_FSRs_to_keys.at(fsr_id))._centroid;
+  }
+  catch(std::exception &e) {
+    log_printf(ERROR, "Could not find centroid in FSR: %i. "
+               "Backtrace:%s", fsr_id, e.what());
```

Comment:
I've tried to remove the backtraces from the error messages, since the logger module throws an exception anyway. This would look like:

```C++
catch(std::exception &e) {
  log_printf(ERROR, "Could not find centroid in FSR: %i.", fsr_id);
```